lib/analyzer/os-release/release-analyzer.ts

```diff
@@ -16,7 +16,16 @@ export async function tryOSRelease(text: string): Promise<OSRelease | null> {
     version = version.split(".")[0];
   }
 
-  return { name, version };
+  let distroless = false;
+  const prettyNameRes = text.match(/^PRETTY_NAME=(.+)$/m);
```

Comment:
what do you think about always returning the entire `PRETTY_NAME`? just in case we'll want that to other use cases as well

Reply:
I think I wish I had thought about it before changing 3 test files 😅